react/modules/searchMetadata.ts

```diff
@@ -122,10 +122,18 @@ export const getSearchMetadata = (searchQuery?: SearchQueryData) => {
     return
   }
 
+  let decodedTerm = ''
+  // This try/catch works to prevent decoding search terms that end in "%".
+  try {
+    decodedTerm = decodeURIComponent(searchTerm)
+  } catch (e) {
+    decodedTerm = decodeURIComponent(encodeURIComponent(searchTerm))
```

Comment:
Isn't that just the same as the following suggestion?

```suggestion
    decodedTerm = searchTerm
```

Reply:
In fact it is, but when I put this I consider that we may have a case where we need to encode and then decode the term, with this in mid do we keep the decode/encode logic or simply pass the term?